com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -124,11 +127,15 @@ CBUFFER_END
 #if defined(USING_STEREO_MATRICES)
 CBUFFER_START(UnityStereoViewBuffer)
 float4x4 unity_StereoMatrixP[2];
-float4x4 unity_StereoMatrixIP[2];
+float4x4 unity_StereoMatrixInvP[2];
```

Comment:
renaming shader variable might break existing shaders. we can "deprecate" old name but still make it define to new name.

Reply:
I added the old symbols in the deprecated.hlsl file.